app/views/editorial/news/new.html.haml

```diff
@@ -0,0 +1,15 @@
+%h1 News article
+
+.fieldset
+  = simple_form_for [:editorial, @form], as: :node do |f|
+    = f.input :section_id, collection: @sections, label: 'Publisher'
+    = f.input :name, label: 'Title'
+    = f.input :short_summary
+    = f.input :summary, as: :text, input_html: {rows: 5}
+    = f.input :content_body, as: :text, label: 'Body', input_html: {rows: 15}
+    = render partial: '/editorial/nodes/news_article', locals: {f: f}
```

Comment:
Does this partial even make sense anymore? It possibly just belongs in this form.